src/bug-filing/null-issue-filing-service/null-issue-filing-service.tsx

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { NamedSFC } from '../../common/react/named-sfc';
+import { BugFilingServiceProviderImpl } from '../bug-filing-service-provider';
+import { BugFilingService } from '../types/bug-filing-service';
+import { SettingsFormProps } from '../types/settings-form-props';
+
+const nullServiceKey = 'none';
+
+const renderSettingsForm = NamedSFC<SettingsFormProps<{}>>('NullIssueFilingService', () => null);
+
+export const NullIssueFilingService: BugFilingService = {
+    key: nullServiceKey,
+    isHidden: true,
+    order: 0,
+    displayName: 'None',
+    renderSettingsForm,
+    buildStoreData: () => null,
+    isSettingsValid: () => false,
+    createBugFilingUrl: () => null,
```

Comment:
This might just be '#'?

Reply:
should never get there... I'm tempted to actually throw an error instead